### ISSUE TITLE
Run pug-lint from `node_modules`

### ISSUE THREAD

**Issue by w0rp** [mrmlnc/vscode-puglint#74]
I would like it if the extension ran `pug-lint` from `node_modules` instead of globally, because this has been my experience.

1. Install the extension.
2. There are no error messages and nothing happens.
3. Install `pug-lint` globally.
4. Now it works.

This means that I have to make sure everyone on my team runs the same `pug-lint` version, and I have to update a wiki telling everyone which version of `pug-lint` to install on their machine. When people forget to do this, they check in code with problems, and don't know about them until the CI server tells them. It's easier if the extension just detects pug-lint from `node_modules`, and runs that.

**Comment by mrmlnc** [comment]
Hello, @w0rp,

You can install `pug-lint` as project dependency to `devDependencies` (more details here: https://github.com/mrmlnc/vscode-puglint#installation). Every developer must install project dependencies to work with it in any case.

**Comment by w0rp** [comment]
Does it have to be listed in `devDependencies` to work? It's listed in `dependencies` at the moment, because we only use Node to build JS for running in browsers.

**Comment by mrmlnc** [comment]
Nope, you can specify a dependency in any fields – the main thing is that the dependency is must be install.

We use the [`npm-module-path`](https://github.com/mrmlnc/npm-module-path) package to find dependency.
